students/kimeunji/westagram/users/views.py

```diff
@@ -1,3 +1,36 @@
-from django.shortcuts import render
+import json
+import re
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from users.models import User
+
+class UserView(View):
+    def post(self, request):
+        try:
+            data        = json.loads(request.body)
+            EMAIL_RE    = r'^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+            PASSWORD_RE = r'^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$'
+            email = data['email']
+            password = data['password']
```

Comment:
```suggestion
            EMAIL_RE    = r'^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
            PASSWORD_RE = r'^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$'
            
            data     = json.loads(request.body)
            email    = data['email']
            password = data['password']
```
다음과 같이 위치 조정이랑, = 기준으로 정렬하는 것이 조금 더 가독성이 좋아 보입니다.